src/components/icons/icon_wrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+export type Size = 'sm' | 'md' | 'lg';
```

Comment:
Maybe call these IconSize and IconColor to prevent polluting the namespace when importing these (too generic).